scripts/mirror.js

```diff
@@ -219,71 +219,61 @@ const bumpChromeAndroid = (originalData, sourceData, source) => {
 };
 
 /**
- * @param {SupportStatement} originalData
- * @param {SupportStatement} sourceData
- * @param {string} source
+ * @param {SupportStatement} comp
  * @returns {SupportStatement}
  */
-const bumpEdge = (originalData, sourceData, source) => {
-  let newData = {};
-
-  if (source == 'ie') {
-    if (sourceData.version_removed && sourceData.version_removed !== null) {
-      newData.version_added = false;
-    } else if (sourceData.version_added !== null) {
-      newData.version_added = sourceData.version_added ? '12' : null;
-    }
+const bumpEdge = comp => {
+  let newData = comp['edge'];
+  let originalData = comp['edge'];
+  let ieData = comp['ie'];
+  let chromeData = comp['chrome'];
 
-    if (sourceData.notes) {
-      newData.notes = updateNotes(
-        sourceData.notes,
-        /Internet Explorer/g,
-        'Edge',
-      );
-    }
-  } else if (source == 'chrome') {
-    newData = originalData == undefined ? sourceData : originalData;
+  if (ieData.version_removed && ieData.version_removed !== null) {
```

Comment:
```suggestion
  if (ieData.version_removed !== null) {
```

This is an equivalent check. But it still looks a bit funny. Should it be `ieData.version_added && !ie_data.removed_removed`?